sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/AgentCoreEntrance.java

```diff
@@ -136,6 +139,20 @@ public static void install(String artifact, Map<String, Object> argsMap, Instrum
         if (NotificationManager.isEnable()) {
             NotificationManager.doNotify(new NotificationInfo(SermantNotificationType.LOAD_COMPLETE, null));
         }
+
+        // cache instrumentation
+        CommandProcessor.cacheInstrumentation(instrumentation);
```

Comment:
i think ExternalAgentManager caching instrumentation is better, and handleAgentInstallation method can reduce an input parameter

Reply:
instrumentation can be use by the second sermant agent install or uninstall in the future，so put it in CommandProcessor